datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -621,11 +621,26 @@ private void uninitializeTask(DatastreamTask t) {
     _cpProvider.unassignDatastreamTask(t);
   }
 
+  private boolean getCustomCheckpointing(DatastreamTask task) {
```

Comment:
nit: might be better to keep `uninitializeTask` and `initializeTask` adjacent.

Reply:
Fixed this.